R/ggplotly.R

```diff
@@ -242,6 +242,41 @@ gg2list <- function(p){
     if (!all(barmodes == barmodes[1]))
       warning(paste0("You have multiple barcharts or histograms with different positions; ",
                      "Plotly's layout barmode will be '", layout$barmode, "'."))
```

Comment:
This warning existed before and I'm not sure of an example where it's applicable, but it might be worth keeping in case we run across an edge case...